sql/log_event.cc

```diff
@@ -7349,7 +7351,8 @@ Rows_log_event::Rows_log_event(THD *thd_arg, TABLE *tbl_arg,
       m_key(NULL),
       m_key_info(NULL),
       m_distinct_keys(Key_compare(&m_key_info)),
-      m_distinct_key_spare_buf(NULL) {
+      m_distinct_key_spare_buf(NULL),
+      master_had_triggers(0) {
```

Comment:
:(
```suggestion
      master_had_triggers(false) {
```
or just use field initializers for both added fields